notebooks/timelapse.ipynb

```diff
@@ -388,7 +360,7 @@
     "cloud_filter = CloudCoverageFilter(max_cc=0.05)\n",
```

Comment:
Maybe we could rename `max_cc` to `maxcc` to be in line with naming in `eo-learn` and `sentinelhub-py`.